plugins/woocommerce/includes/class-wc-frontend-scripts.php

```diff
@@ -492,6 +492,7 @@ private static function get_script_data( $handle ) {
 							'smoothHeight'   => true,
 							'directionNav'   => false,
 							'controlNav'     => 'thumbnails',
+							'manualControls' => apply_filters( 'woocommerce_single_product_nav_flexslider', get_theme_support( 'wc-product-gallery-slider-nav' ) ) ? '.flex-control-nav img' : null,
```

Comment:
(I premise that creating the docblock looks right to me) I can't find even one similar as an example.... eg. "woocommerce_single_product_flexslider_enabled" or anyone of [this](https://github.com/woocommerce/woocommerce/blob/b2392036eeee949ee7e7c2130578d38f237c8baf/plugins/woocommerce/includes/class-wc-frontend-scripts.php#L495-L515). 

The Code sniff was triggered for Line 58 but is something i haven't touched/modified

Reply:
Yep, we can ignore code sniffs for lines that are untouched (that's just an automation trying to be helpful). You're also correct that there are many existing hooks that are not documented—but we want to avoid that with any new code.

In this case, adding a docblock would look something like this:

```php
						/**
						 * Controls if manual controls for the single product flexslider should be enabled.
						 *
						 * @since 6.7.0
						 * 
						 * @param bool $enable Enable manual controls.
						 */
						'manualControls' => apply_filters( 'woocommerce_single_product_nav_flexslider', get_theme_support( 'wc-product-gallery-slider-nav' ) ) ? '.flex-control-nav img' : null,
```

So, right there in the array, immediately preceding the `apply_filters()` call.

Reply:
(...and the same principle extends to any other new or modified hooks.)